src/decorators/restrict.js

```diff
@@ -9,3 +9,6 @@ export default function restrict(func) {
 		return descriptor
 	}
 }
+
+export const needPermissions = (permissions, ...args) =>
+	restrict(({ member }) => member && member.hasPermission(permissions, ...args))
```

Comment:
why don't you just do this
```js

Reply:
Idk xD, I just copied the arguments in the original `needPermissions`